### PR TITLE
Rename the ClowdApp to kessel-relations-api

### PR DESCRIPTION
**Pull request by gwenneg**
## Describe your changes

Related to https://github.com/project-kessel/inventory-api/pull/110

- This PR renames pods in OpenShift from `relations-relations` to `kessel-relations-api` and `relations-spicedb-spicedb` to `kessel-relations-spicedb-spicedb`. I'm not sure where the extra `-spicedb` comes from, so it'll remain in the name for now.

## Ticket reference (if applicable)
Fixes #

## Checklist

* [ ] Are the agreed upon acceptance criteria fulfilled?

* [ ] Was the 4-eye-principle applied? (async PR review, pairing, ensembling)

* [ ] Do your changes have passing automated tests and sufficient observability?

* [ ] Are the work steps you introduced repeatable by others, either through automation or documentation?
  * [ ] If automation is possible but not done due to other constraints, a ticket to the tech debt sprint is added
  * [ ] An SOP (Standard Operating Procedure) was created

* [ ] The Changes were automatically built, tested, and  - if needed, behind a feature flag - deployed to our production environment. (**Please check this when the new deployment is done and you could verify it.**)

* [ ] Are the agreed upon coding/architectural practices applied?

* [ ] Are security needs fullfilled? (e.g. no internal URL)

* [ ] Is the corresponding Ticket in the right state? (should be on "review" now, put to done when this change made it to production)

* [ ] For changes to the public API / code dependencies: Was the whole team (or a sufficient amount of ppl) able to review?

### CODE REVIEW

File: deploy/kessel-relations-deploy.yaml [hunk 1]
```diff
@@ -36,13 +36,12 @@ objects:
     kind: ClowdApp
     metadata:
       name: ${CLOWDAPP_NAME}
-      namesapce: ${NAMESPACE}
```
Comment (gwenneg) [inline deploy/kessel-relations-deploy.yaml:39]: Not only was that not needed, but the key also contained a typo.

File: deploy/kessel-relations.yaml [hunk 2]
```diff
@@ -147,12 +147,9 @@ objects:
               enabled: true
               apiPath: authz
 parameters:
-  - description: Name of the ClowdApp
-    name: CLOWDAPP_NAME
```
Comment (gwenneg) [inline deploy/kessel-relations.yaml:151]: This wasn't used.

File: deploy/kessel-relations-deploy.yaml [hunk 3]
```diff
@@ -74,13 +73,10 @@ objects:
...
   - description: Name of the ClowdApp
     name: CLOWDAPP_NAME
-    value: relations
-  - description: ClowdEnvironment name
+    value: kessel-relations
+  - description: ClowdEnvironment name (ephemeral, stage, prod)
     name: ENV_NAME
-    value: ${ENV_NAME}
-  - description: Namespace to deploy into
-    name: NAMESPACE
```
Comment (gwenneg) [inline deploy/kessel-relations-deploy.yaml:82]: This is no longer needed. To be removed from app-interface.

File: deploy/kessel-relations-deploy.yaml
Comment (gwenneg) [inline deploy/kessel-relations-deploy.yaml:1]: This file is used in stage.

File: deploy/kessel-relations.yaml
Comment (gwenneg) [inline deploy/kessel-relations.yaml:1]: This file is used in ephemeral.

File: deploy/kessel-relations-deploy.yaml [hunk 4]
```diff
@@ -32,13 +32,12 @@ objects:
...
     metadata:
       name: ${CLOWDAPP_NAME}
-      namesapce: ${NAMESPACE}
     spec:
       envName: ${ENV_NAME}
       testing:
         iqePlugin: relations_api
       deployments:
-        - name: relations
+        - name: api
```
Comment (Rajagopalan-Ranganathan) [inline deploy/kessel-relations-deploy.yaml:40]: api seems to generic - should we rename it to relations-api. given inventory and other components are there as a part of kessel? WDYT?

File: deploy/kessel-relations.yaml [hunk 5]
```diff
@@ -107,13 +107,13 @@ objects:
...
     metadata:
-      name: relations
+      name: kessel-relations
     spec:
       envName: ${ENV_NAME}
       testing:
         iqePlugin: relations_api
       deployments:
-        - name: relations
+        - name: api
```
Comment (Rajagopalan-Ranganathan) [inline deploy/kessel-relations.yaml:116]: same as above relations-api?

File: deploy/kessel-relations-deploy.yaml (hunk 4 above)
Comment (gwenneg) [reply deploy/kessel-relations-deploy.yaml:40]: The pods will appear as `ClowdApp.name + "-" + Deployment.name` in OpenShift, so `kessel-relations-api` in this case. `api` is only the name of one of the deployments from the `relations-kessel` ClowdApp.

Comment (gwenneg) [reply deploy/kessel-relations-deploy.yaml:40]: FYI the exact same naming convention is already used for `inventory-api` in the kessel-stage namespace, if you want to see how it looks.

Comment (Rajagopalan-Ranganathan) [reply deploy/kessel-relations-deploy.yaml:40]: perfect thanks for the clarification

**Review by Rajagopalan-Ranganathan — APPROVED** [review]

**Pull request merged**